.github/workflows/fprime-gds-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: fprime-gds tests
+
+on: [pull_request]
```

Comment:
Can you please make this workflow triggered on both pull requests and pushes (so that it runs on the devel branch as well)